exchange/price_granularity.go

```diff
@@ -1,20 +1,33 @@
 package exchange
 
 import (
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/openrtb_ext"
 	"math"
 	"strconv"
-
-	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
 // GetPriceBucket is the externally facing function for computing CPM buckets
-func GetPriceBucket(cpm float64, config openrtb_ext.PriceGranularity) string {
+func GetPriceBucket(bid openrtb2.Bid, targetingData *targetData) string {
 	cpmStr := ""
 	bucketMax := 0.0
 	bucketMin := 0.0
 	increment := 0.0
+
+	config := targetingData.priceGranularity //assign default price granularity
+	if targetingData.mediaTypePriceGranularity != nil {
+		bidType, err := getMediaTypeForBid(bid)
+		if err != nil {
+			config = targetingData.priceGranularity //assign default price granularity
```

Comment:
I put it here to show explicitly we want to use default price granularity in case of error - if media type for bid is not found. Otherwise I'll need to ignore the error. 
I modified comment. 
I'm open to change it if you think it should be different.